lib/data/ncsu/traject_config.rb

```diff
@@ -69,12 +69,12 @@ def item_status(current, home)
     end
     # $k is only present if current != home
     # needs refinement for reserves etc. and non-lending items
-    current = item['current_location']
-    home = item['shelving_location']
+    current = item['loc_current']
+    home = item['loc_n']
     item['status'] = item_status(current, home)
 
-    if item.fetch('call_number_scheme', '') == 'LC'
-      lcc_top.add(item['call_number'][0, 1])
+    if item.fetch('cn_scheme', '') == 'LC'
+      lcc_top.add(item['call_no'][0, 1])
     end
```

Comment:
Do we need item.delete('loc_current') before the next line?

Reply:
If loc_current isn't needed in the output then yes it can be deleted from the hash before it gets accumulated.

Reply:
We hadn't defined a loc_current item key for Argot (OMG another kind of location!) so I was assuming it was just being used to determine status... @adjam can you confirm?

Reply:
@corylown @kspurgin lcc_top can go away
loc_current is local to NCSU, it's how we compute availability.  don't touch it!

Reply:
OK -- I think this can get merged as is then.